src/sentry/api/bases/organization_events.py

```diff
@@ -48,8 +48,7 @@ def get_snuba_query_args_v2(self, request, organization, params):
         except InvalidSearchQuery as exc:
             raise OrganizationEventsError(exc.message)
 
-        fields = request.GET.getlist('fields')[:]
-
+        fields = request.GET.getlist('field')[:]
```

Comment:
Do we use `field` or `fields` in discover? It would be nice to have as much consistency as we can so that both internal and external folks don't have to remember two sets of names for APIs that function in similar ways.

Reply:
It's `fields` in Discover, but the format is quite different there, and my thought was to use the singular here in order to match all the other query params for this endpoint.